assets/README

```diff
@@ -412,6 +412,28 @@ The "make install" command will create these database related scripts in
 done at configure time is successfull ("database schema create" reported
 as yes at the end of the configure script execution)
 
+
+7-2 - UPDATING FROM A TANGO 9.2.5 DATABASE
```

Comment:
Very strange section number (was already like that before)...
I would start with 7-1 and then change the numbers of the following 7-x sections (7-2, 7-3, etc...).

Reply:
Thanks! Fixed as proposed.